src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -27,64 +27,64 @@ showStateDialogTitle = Zustand für %s
+# ==> fineDetail = Capture fine-grained details
+# ==> stepFineDescription = Data is captured whenever any signal value changes, including any transient signal fluctuations that may happen during propagation. Transient fluctuations are recorded as if the gate delay were %s, and stable signals are recorded as if they were held stable for %s.
+realTime = Durchgehender Echtzeit-Modus
+# ==> realCoarse = Coarse-grained summary
+# ==> realCoarseDescription = Data is captured continuously, whenever the simulator is enabled, but transient signal fluctuations are ignored. Each second of real time is recorded as %s of simulated circuit time.
+# ==> realFine = Fine-grained details
+# ==> realFineDescription = Data is captured continuously, whenever the simulator is enabled, includin transient signal fluctuations. Each second of real time is recorded as %s of simulated circuit time.
+# ==> clockTime = Clocked mode
+# ==> clockCoarse = Coarse-grained summary
+# ==> clockCoarseDescriptionDual = Data is captured at each clock transition, ignoring any transient fluctuations that may happen between transitions or during propagation. Each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionRising = Data is captured at each rising clock transition, ignoring any transient fluctuations that may happen between transitions or during propagation. Each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionFalling = Data is captured at each falling clock transition, ignoring any transient fluctuations that may happen between transitions or during propagation. Each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionHigh = Bei Hohem pegel des Taktsignales werden Daten aufgenommen. Änderungen werend der Pegel niedrig ist werden Ignoriert. Changes when the clock is high are recorded as if the gate delay were %s, and each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionLow = Data is captured continuously only when the clock is low, ignoring any transient signal fluctuations that may happen during propagation or while the clock is high. Changes when the clock is low are recorded as if the gate delay were %s, and each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockFine = Fine-grained details
+# ==> clockFineDescription = Data is captured whenever any signal value changes, including any transient signal fluctuations that may happen during propagation. Transient fluctuations are recorded as if the gate delay were %s, and each clock cycle is recorded as if it took %s per cycle (%s per tick).
+clockSourceLabel = Takt Quelle:
+clockSourceNone = nicht verfügbar
+selectClockDriverMessage = Diese Funktion braucht ein Tacktsignal, aber in der Schaltung wurde keines gefunden. Fügen sie ein Taktsignal ein oder wählen sie eines der Signale as Takt.
```

Comment:
"Diese Funktion braucht ein Taktsignal, aber in der Schaltung wurde keines gefunden."